app/src/androidTest/java/net/skyscanner/backpack/compose/bottomnav/BpkBottomNavTest.kt

```diff
@@ -31,16 +32,16 @@ class BpkBottomNavTest : BpkSnapshotTest() {
 
   @Before
   fun setup() {
-    setDimensions(height = 80, width = 250)
+    snapshotSize = IntSize(250, 80)
```

Comment:
I'm wondering whether it's possible to have wrapContent size by default – in that case we wouldn't need to worry about estimating the size in 99% of the cases

Reply:
yeah i can take a look at that. but would do it outside of this PR too